### PR TITLE
Index fix

### PR DESCRIPTION
**Pull request by robogen**
This fixes a slight issue on the index.html page so the 'c3' and 'd3' libraries are included through the node_modules folder instead of the lib folder.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ module.exports = function(grunt) {
 	  stdout: true,
 	  stderr: true,
 	  failOnError: true,
-	  npm: '--production'
+	  npm: '--development'
```
Comment (djw8605) [inline Gruntfile.js]: What does this change?

File: HCCGo/app/index.html [hunk 2]
```diff
@@ -15,7 +15,7 @@
     <link href="lib/bootstrap/css/bootstrap.css" rel="stylesheet">
     <link href="lib/bootstrap-material-design/material.css" rel="stylesheet">
     <link href="lib/bootstrap-material-design/ripples.css" rel="stylesheet">
-    <link href="lib/c3/c3.css" rel="stylesheet">
+    <link href="../node_modules/c3/c3.css" rel="stylesheet">
```
Comment (djw8605) [inline HCCGo/app/index.html]: I'm worried, are these guaranteed to be available to the browser?  Seems odd to go up a directory.

File: Gruntfile.js (hunk 1 above)
Comment (robogen) [inline Gruntfile.js]: Ah, right! This affects how the devDependencies in package.json get used, '--production' ignores them and '--development' ensures they're present. The difference also affects packaging, where with '--development' the packages listed under devDependencies get packaged with the application and with '--production' they get ignored and left out.

Comment (djw8605) [inline Gruntfile.js]: So, should we have it as development?

File: HCCGo/app/index.html (hunk 2 above)
Comment (robogen) [inline HCCGo/app/index.html]: Yes, these should remain available, as the hierarchy doesn't change after packaging. And it does seem a little odd but the directory above this one is where the node_modules folder needs to be.

File: Gruntfile.js (hunk 1 above)
Comment (robogen) [inline Gruntfile.js]: While developing code, production when packaging.

**Pull request merged**